# GRCh38 (also GRCh38.p13)

## URL: https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/001/405/GCA_000001405.29_GRCh38.p14/

1. the path is all-GCA-000-001-405 for all the different version of human reference genome
1. Check other number-file locations and correspond to other stuff.
1. Find sequence by chromosome or whole reference sequence in one file
1. Find the polymorphisms and programmatically generate the hashes.
1. Carry out a graphical/verbal representation of the data.
1. Create command line type commands that can be chained together.
1. Use default input FASTA format or flat text file.

## URL https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/001/405/GCA_000001405.29_GRCh38.p14/GCA_000001405.29_GRCh38.p14_assembly_structure/Primary_Assembly/assembled_chromosomes/FASTA/


## What I remember from before
1. Protein sequences and cDNA sequences are only about 3 percent. 
1. There two big repositories with genomic data: ncbi and ensemble.
1. Not sure what are the differences yet. 
1. In Ensemble: the sequence label has many descriptors, not sure if ncbi has the same.
1. I need to compare how related are whole genome sequence in one file, or per chromosome, 
1. Also how the cDNA sequences in one file or per chromosome are related. 
1. In the full cDNA sequences few of them where repeated and with different labels, why?
1. Find them again and systematize the finding
1. explore the graphical interfase, I tried using plain HTML/CSS but not reliable.
1. Explore three or any other graphical interfase for more reliable and long lasting results



## Use chromosome 22 as case study, then 1 and Y

1. Extract the files in extracted_files folder
1. review reference coordinates and how SNPs are referenced
1. search the web for etnicity information (and possibly any other useful information) based on reported sequences and reported SNPs. 
1. I assume all the SNPs are already verified and (..sponsored reviewed by a human)
1. In the se